raphtory/src/db/api/view/time.rs

```diff
@@ -280,6 +330,35 @@ impl<'graph, V: OneHopFilter<'graph> + 'graph + InternalTimeOps<'graph>> TimeOps
             _ => WindowSet::empty(parent),
         }
     }
+
+    fn rolling_aligned<I>(
+        &self,
+        window: I,
+        step: Option<I>,
+    ) -> Result<WindowSet<'graph, Self>, ParseTimeError>
+    where
+        Self: Sized + Clone + 'graph,
+        I: TryInto<Interval>,
+        ParseTimeError: From<<I as TryInto<Interval>>::Error>,
+    {
+        let parent = self.clone();
+        match (self.timeline_start(), self.timeline_end()) {
```

Comment:
If the graph is explicitly windowed, we probably don't want to do alignment if we rename this back to `rolling`. The easiest way to achieve that would be to pass the alignment unit as input to `timeline_start` and `timeline_end` which can handle that.